dbmail/backends/mail.py

```diff
@@ -70,7 +70,7 @@ def _insert_mailer_identification_head(self):
     def _get_connection(self):
         if self._template.auth_credentials:
             return self._kwargs.pop('connection', None) or get_connection(
-                **self._template.auth_credentials)
```

Comment:
dbmail.models.MailTemplate#get_template

    obj.__dict__['auth_credentials'] = auth_credentials

why you are change ``**self._template.from_email.get_auth()`` instead cached ``auth_credentials``?

and we are working with development branch. please change code at dev branch, and resent pull request.